scalability_and_performance/cnf-numa-aware-scheduling.adoc

```diff
@@ -8,8 +8,12 @@ toc::[]
 
 Learn about NUMA-aware scheduling and how you can use it to deploy high performance workloads in an {product-title} cluster.
 
-:FeatureName: NUMA-aware scheduling
-include::snippets/technology-preview.adoc[leveloffset=+1]
```

Comment:
If not mistaken `technology-preview.adoc` is a file , and since the old TP note is not relevant anymore, can we remove this file completly from the 4.12 docs branch to avoid confusion in future doc modifications?

Reply:
This file is re-used elsewhere in the documentation so it shouldn't be removed.